webapp/app.py

```
@@ -779,5 +779,63 @@ def language(lang_code):
+            n=1,
+        )
+
+        image_url = response.data[0].url
+        if not image_url:
+            return "Image generation failed", 500
+
+        # Download and cache the image
+        import urllib.request
+
+        os.makedirs(cache_dir, exist_ok=True)
+        urllib.request.urlretrieve(image_url, cache_path)
```

Ruff (0.15.1)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)